### ISSUE TITLE
[Bug]: The default thumbnail preference in Flatpak version is `hidden`

### ISSUE THREAD

**Issue by M1ke5225** [FreeTubeApp/FreeTube#7280]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

When Freetube is installed from Flathub on Fedora, the default *Thumbnail Preference* is `hidden`.

### Expected Behavior

The default should be the `Default` value.

### Issue Labels

accessibility issue

### FreeTube Version

Version 0.23.4 Beta

### Operating System Version

Fedora release 42 (Adams)

### Installation Method

Flathub

### Primary API used

Local API

### Additional Information

This issue has been present in at least two earlier Freetube versions as well.

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).
